website/docs/policies/expression.mdx

```diff
@@ -54,6 +54,11 @@ import Objects from "../expressions/_objects.md";
     -   `request.context`: A dictionary with dynamic data. This depends on the origin of the execution.
 
 -   `geoip`: GeoIP object, see [GeoIP](https://geoip2.readthedocs.io/en/latest/#geoip2.models.City)
+
+    ```python
+    return context["geoip"].country.iso_code == "US"
+    ```
+
 -   `ak_is_sso_flow`: Boolean which is true if request was initiated by authenticating through an external provider.
```

Comment:
@Rukk385 and @rissson, does this rewrite keep the technical accuracy? "Boolean. By default, this flag is set to True if request was initiated by authenticating through an external provider."

We need to separate the fact that it is boolean from rest of info. And is it a flag, or a value? Finally, does the system set this value (by default), or the user?

Reply:
This is out of scope for this PR I think.